mlflow/cli/__init__.py

```diff
@@ -467,6 +478,29 @@ def server(
         uvicorn_opts=uvicorn_opts,
     )
 
+    if disable_security_middleware:
```

Comment:
Since we use FastAPI middleware to implement security measures, should we raise if Flask is directly used with security-related parameters in `_validate_server_args`?